components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -111,6 +111,14 @@ class BrowserToolbar @JvmOverloads constructor(
         get() = display.siteSecurity
         set(value) { display.siteSecurity = value }
 
+    override var sitePermission: Toolbar.SitePermission = Toolbar.SitePermission.NONE
```

Comment:
nit: In the state we talk about `PermissionHighlights` and here we are using `SiterPermission`. I wonder if there's an opportunity to unify the name? Or do we want to keep separate namings?

Reply:
Yes, we can have the same naming 👍🏽